Javascript Cabstone project

Mo7amedSMahdi opened the pull request:
Add mobile home page.
Add mobile about page.
Add desktop version for both Home and about pages.
Add Speakers dynamically to the home page.
Add mobile menu.

check out the video that describe the project
https://www.loom.com/share/deb9c019819043608e556c75b68e2b3a

Mo7amedSMahdi committed:
staging

Mo7amedSMahdi committed:
Merge remote-tracking branch 'origin/main' into main

Mo7amedSMahdi committed:
Add the Full frontend desktop, Tablet and mobile site.

Mo7amedSMahdi committed:
Create about page and Javascript

- Add about page content.
- Add css styles for the about page
- Add speakers dynamically using Javascript to the index page.
- Add functional mobile menu.
- Add descriptive README for the project.

Mo7amedSMahdi committed:
Update README.md

Mo7amedSMahdi committed:
Change the screenshots images

Mo7amedSMahdi committed:
Update README.md

Mo7amedSMahdi committed:
fix linters

Mo7amedSMahdi committed:
Merge branch 'about-page' of https://github.com/Mo7amedSMahdi/JavaScript-Capstone-Project into about-page

elmar8287 commented on README.md:
```diff
@@ -1 +1,90 @@
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Ckeckout The live Site here](https://mo7amedsmahdi.github.io/JavaScript-Capstone-Project/)
+
```
- [x] Please, consider fixing the demo live version - it is actually not the last updated one:
<img width="1035" alt="Screen Shot 2022-03-12 at 22 33 59" src="https://user-images.githubusercontent.com/49064106/158030477-c6b8dd75-0bb1-4864-afe3-196efdee7aca.png">

elmar8287 commented on about.html:
```diff
@@ -5,10 +5,120 @@
 
 <body>
+    <header class="header column">
+        <div class="header__nav flex bg-dark">
+            <a href="#"><img src="images/icons/facebook.svg" alt="facebook" /></a>
+            <a href="#"><img src="images/icons/twitter.svg" alt="twitter" /></a>
+            <div class="button-group flex">
+                <button type="button" class="btn btn-text">English</button>
+                <button type="button" class="btn btn-text">My Page</button>
```
- [x] I have investigated, why is the first link line (the dark one) is not displaying: the reason is, that you have given for class `header__nav` display: none; You have to set it for the mobile screen, but for the desktop version, it should not be `none` - I know you can fix it easily 👍 

> Original design
<img width="986" alt="Screen Shot 2022-03-12 at 22 37 12" src="https://user-images.githubusercontent.com/49064106/158030946-6dff4cd5-4f85-46fb-b2b0-55c65b988437.png">

Your version:
![2022-03-12 22 50 36](https://user-images.githubusercontent.com/49064106/158030999-ccd9ccf1-3037-46f6-981c-11d9b1b6bc69.gif)

elmar8287 commented on index.html:
```diff
@@ -5,46 +5,52 @@
-                <p>@ National Musem of Korea, Art Center Nabi and more</p>
+                <h2>2019.10.15(THU) ~ 16(FRI)</h2>
+                <p>@Al-shaab Stadium</p>
             </div>
```
- [x] Please, consider fixing the position of content in your hero section, like in the original design. 
- [x] Please, consider using the same color and fonts, as in the original design.

> Your version: #1 Try to move it right, under the logo 👍 
<img width="855" alt="Screen Shot 2022-03-12 at 23 04 25" src="https://user-images.githubusercontent.com/49064106/158031751-5ca1c4c2-0a55-4a4e-b7fe-cf68bd0abf09.png">

> Original design:
<img width="748" alt="Screen Shot 2022-03-12 at 22 52 40" src="https://user-images.githubusercontent.com/49064106/158031778-e068af1e-a41e-409e-b02f-5fb797e1a5b0.png">

> The color and fonts requirements:
<img width="1085" alt="Screen Shot 2022-03-12 at 23 13 55" src="https://user-images.githubusercontent.com/49064106/158031799-56831d38-e81e-45f4-8825-44c13581959a.png">

elmar8287 commented on index.html:
```diff
@@ -53,130 +59,67 @@ <h2 class="section--title">Main Program</h2>
             <div class="program__group flex">
                 <div class="program">
                     <i class="icon icon-lecture"></i>
```
- [ ] Please, consider fixing this "Main program" section. Here you have to consider the original design. Please, follow the instruction on screenshots.

> Original design:
<img width="963" alt="Screen Shot 2022-03-12 at 23 16 50" src="https://user-images.githubusercontent.com/49064106/158032009-56c1d37f-dfa6-4733-8d59-5ba9b8253832.png">

> Your version:
<img width="1267" alt="Screen Shot 2022-03-12 at 23 17 06" src="https://user-images.githubusercontent.com/49064106/158032027-781fd689-619b-43b4-ada2-815052aeb37a.png">

elmar8287 commented on index.html:
```diff
@@ -53,130 +59,67 @@ <h2 class="section--title">Main Program</h2>
-                            movement.</p>
-                    </div>
-                </div>
+            <div id="speakers" class="speakers__group grid">
             </div>
```
- [x] Please, consider fixing the mobile view of the speaker's section: it should display 2 cards and then the "More" button, which will open all other speakers' cards.

> The original design:
<img width="462" alt="Screen Shot 2022-03-12 at 23 46 06" src="https://user-images.githubusercontent.com/49064106/158032970-7980131b-1616-432b-ac69-e3cb54c343e0.png">

> Your version:
![2022-03-12 23 45 45](https://user-images.githubusercontent.com/49064106/158032987-3d1433f9-ea45-41f1-8221-4369dda89095.gif)

elmar8287 commented on about.html:
```diff
@@ -5,10 +5,120 @@
+                </div>
+            </div>
+        </section>
+        <section class="partners flex column bg-dark" id="partners">
+            <h2 class="section--title">Partners</h2>
+            <ul class="nav--brands">
+                <li><img src="images/brands/careem.svg" alt="mozilla co. brand logo"></li>
+                <li><img src="images/brands/google.svg" alt="google co. brand logo"></li>
+                <li><img src="images/brands/naver.svg" alt="naver co. brand logo"></li>
+                <li><img src="images/brands/daumkakao.svg" alt="daumkakao co. brand logo"></li>
+                <li><img src="images/brands/airbnb.svg" alt="airbnb co. brand logo"></li>
+            </ul>
+        </section>
+    </main>
+
+    <footer class="footer flex">
+        <div class="footer__group">
+            <img src="images/brands/it_live.png" alt="creative commones korea logo">
+        </div>
+        <div class="footer__group span-2">
+            <p>"Like Never Before"</p>
+            <p>2019 The largest Iraqi forum for information technology in all its disciplines.</p>
+        </div>
+    </footer>
+
+    <script src="main.js"></script>
```
- [ ] Please, consider removing the partner's section here, as it is in the original design.
- [ ] Please, consider that on the about page the footer should be dark, as it is required.

> Your version:
<img width="1409" alt="Screen Shot 2022-03-13 at 00 07 24" src="https://user-images.githubusercontent.com/49064106/158033414-77a85b1f-92da-415a-8430-9d584b729684.png">

> Original version:
<img width="863" alt="Screen Shot 2022-03-13 at 00 11 57" src="https://user-images.githubusercontent.com/49064106/158033431-ab000b80-d1ab-41f1-8ecd-09ba797ca097.png">

elmar8287 commented on index.html:
```diff
@@ -5,46 +5,52 @@
+        <nav class="nav__main flex">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
+                    class="icon-close"></i></button>
+            <ul id="mobileNav" class="nav--mobile hidden">
```
- [ ] Please, consider fixing the mobile menu: there is an issue when the mobile menu is open and while switching to the desktop version, there is the `x` (close icon) still displaying - I know, you can easily fix it 👍 
![2022-03-13 00 15 14](https://user-images.githubusercontent.com/49064106/158033581-a33c4a90-76a8-4fb8-9831-a797faa1c8bb.gif)

> 
<img width="1052" alt="Screen Shot 2022-03-13 at 00 15 29" src="https://user-images.githubusercontent.com/49064106/158033595-4608e11a-efcb-4810-9ae6-afc19b53c20f.png">

elmar8287 reviewed (changes requested):
Hi, @Mo7amedSMahdi.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I want to highlight the following:

- No linter issues 💯 
- Nice job with building the required pages and choosing the topic of your project 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] I watched the video, you recorded for the presentation of the project: the nice job is done! 👍 But, I kindly ask you to mention in your video a short time for noting the coding part. As you can see, during the whole video you are showing the pages in both desktop and mobile versions, but it will be better to present your code also, by highlighting a piece of the code you find interesting or proud of making.

## Optional suggestions

- For the future, please, consider giving a title for all your commits with capital letter 👍 It will show your commit history more professional 💯 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mo7amedSMahdi committed:
Chages required by code reviewer.

-fix header__nav in tablet and disktop screen sizes.
- Change the padding and font family and font colors.
- Change the background of programs and add borders on hover.
- Change programs button on mobile.
- Show only two speakers on mobile and add button to show more.
- Remove partners from about page on disktop and tablet;
- Change footer background color on desktop and tablet.

medaminedev66 commented on README.md:
```diff
@@ -1 +1,90 @@
+
+## Live Demo
+
+[Ckeckout The live Site here](https://mo7amedsmahdi.github.io/JavaScript-Capstone-Project/)
```
- [ ] The demo link still shows the old version of your website. kindly deploy it again with the new branch so your demo link shows your current work.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/77190203/158038326-312f9a6c-a5c4-4179-9f1b-94624b587ae6.png">

medaminedev66 commented on about.html:
```diff
@@ -5,10 +5,120 @@
+        <section class="partners flex column bg-dark" id="partners">
+            <h2 class="section--title">Partners</h2>
+            <ul class="nav--brands">
+                <li><img src="images/brands/careem.svg" alt="mozilla co. brand logo"></li>
```
- [ ] An image is broken on your design, It would be better to fix it so your design is professional. try to change `careem.svg` to `Careem.svg`.

<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/158038395-528de812-f808-4bab-88de-12c9f098dfde.png">

medaminedev66 reviewed (changes requested):
Hi @Mo7amedSMahdi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mo7amedSMahdi committed:
change brand image alt from mozilla to careem

Mo7amedSMahdi committed:
fix careem image

Mo7amedSMahdi committed:
add footer to main css styles

Mo7amedSMahdi committed:
Merge branch 'schedule-page' into about-page

Mo7amedSMahdi committed:
fix program border hover
